tests/test_orbit_phase.py

```diff
@@ -69,6 +71,16 @@ def test_J1855_zero_ecc(self):
         phs3 = self.mJ1855.orbital_phase(self.mJ1855.T0.value + 0.1, anom="true")
         assert phs3 == phs1, "True anom != Mean anom"
 
+    def test_J1855_ell1(self):
+        log = logging.getLogger("test_J1855_ell1")
```

Comment:
What does this line accomplish? The variable `log` appears not to ever be used.

Reply:
Truthfully, I have no idea.  I was just following what is done in many other test files.  Anyone know?

Reply:
I'm pretty sure that many other files have done what you did. According to the documentation getLogger does not have side effects, so this is a pointless no-op. But if one wanted to log something during the tests one could use the `log` variable and arrange for... something? to happen. But really `pytest` keeps things like print statements together and shows them only if the test fails so I don't see the point of this. Possibly it predates the use of `pytest` and is entirely vestigial?

Reply:
`pytest` has nicer facilities to support test fixtures that make it easier to ensure they are only constructed if needed, so a sweep through the test cases to modernize and simplify them would probably produce a low-effort speedup in the testing runtime. Not relevant to this PR especially though perhaps to #817 .

Reply:
Yeah, I think that log stuff must be vestigial.  I support cleaning up the test cases, since when I make a new one I'd like to have a template to work from, but there are many different ones in our suite and I don't know which template is preferred.

Reply:
I'd be happy to convert a test case or two to a modern pytest-based style, if I had one where I knew what was being tested so I could be sure I wasn't missing the point. Pointing to these in the docs ("how to write a test case") might not be a bad idea either.

Reply:
Thanks, that would be enormously helpful! How about test_pulse_number.py. It was written to check that `Residuals` respect pulse numbers, but it could be extended to include tests to show that Fitters also handle them (as was apparently missed previously!)

Reply:
Well, uh, funny you should say that, #814 has some modern tests I wrote in `test_pulse_number.py`: https://github.com/nanograv/PINT/pull/814/files#diff-b7b91b052c8f8a2c7e59db5eab7db1e0a88093116baf523c0bcc58d518c2f68cR48

Reply:
I could produce a more whiz-bang version, or a simpler cleaner version, if we were going to use it as an example.

Reply:
The most important target audience is a person submitting a PR that knows they need to add a unit test, but isn't sure just how to do it. So, clean and simple is my preference.

Reply:
A couple things:

1. That test imports pytest, but not "unittest" and the test classes don't inherit from `unittest.TestCase`. I don't know what unittest is or if we should be using it.
1. It would be great if in the example where fancy @pytest.mark stuff is done, it has a 1-line comment explaining what that mark is doing (and maybe even a link to the docs on that topic).

Reply:
Logging lines are gone from this pull request